src/languages/en.js

```diff
@@ -325,6 +325,7 @@ export default {
         enterDefaultContact: 'Enter your default contact method',
         defaultContact: 'Default contact method:',
         okayGotIt: 'Okay, Got it',
+        unableToCloseAccount: 'Unable to close account',
```

Comment:
NAB: This could be named to `closeAccountError` simply because everything else is named with prefix the "closeAccount".


> closeAccountWarning: 'Closing your account cannot be undone.',
        closeAccountPermanentlyDeleteData: 'This will permanently delete all of your un...
        closeAccountSuccess: 'Account closed successfully',
        closeAccountActionRequired: 'Looks like you need to complete some actions before cl...
        closeAccountTryAgainAfter: 'and try again after.',

Reply:
I agree, let's change this for consistency! @sobitneupane